widget/edit.html

```diff
@@ -10,94 +10,104 @@ <h3 class="modal-title col-md-9">Record CTA Block Edit View</h3>
     </div>
```

Comment:
line no.2 - data-ng-class="{'state-wait': processing }" - processing variable is not define in edit.js so no use of this statement. Please verify.